tomviz/WebExportWidget.h

```diff
@@ -18,6 +18,9 @@
 
 #include <QDialog>
 
+#include "PythonUtilities.h"
+#include "Utilities.h"
+
```

Comment:
This is the section I don't like @cjh1 but I don't know how to write it in a better way.